3529/2/Lib/context_handler.rb

```diff
@@ -0,0 +1,18 @@
+require './Lib/Utiles/selfie_handler.rb'
+require './Lib/checkin_context.rb'
+require './Lib/checkout_context.rb'
+
+module ContextHandler
+  include SelfieHandler
+  include StartContext
```

Comment:
why we need this includes?

Reply:
You mean that i should put all methods from modules just in this file?